src/core/builder/hardhat.ts

```diff
@@ -14,17 +16,43 @@ export class HardhatBuilder implements IBuilder {
   }): Promise<{
     contracts: ContractPayload[];
   }> {
+    // TODO this will work for .js but we also need to handle .ts cases
+    let hardhatConfig: HardhatUserConfig = {
+      paths: {
+        artifacts: "./artifacts",
+        sources: "./contracts",
+      },
+    };
+    try {
+      const readHardHatConfig: HardhatUserConfig = require(join(
```

Comment:
I assume this is how hardhat loads its own config file? What do they do for .ts ?